testnets/testnets.md

```diff
@@ -40,50 +40,84 @@ wasmcli query supply total
 wasmcli query staking validators
 wasmcli query wasm list-code
 
-# create validor address
-wasmcli keys add validator
+# create wallet
+wasmcli keys add mywallet
 ```
 
+In order to join the network as validator, you need some staking tokens. We have a faucet running for that.
+
 Request some tokens from faucet:
 
 ```sh
-export VAL_ADDR=$(wasmcli keys show -a validator)
+# copy the address and paste it to ${WALLET_ADDR} below
+wasmcli keys show -a mywallet
+
+# get some hot stake
+curl --header "Content-Type: application/json" \
+  --request POST \
+  --data '{"ticker":"STAKE","address":"${WALLET_ADDR}"}' \
+  https://$FAUCET/credit
 
+# get coins for paying fees
 curl --header "Content-Type: application/json" \
   --request POST \
-  --data '{"ticker":"STAKE","address":"${VAL_ADDR}"}' \
-  $FAUCET/credit
+  --data '{"ticker":"COSM","address":"${WALLET_ADDR}"}' \
+  https://$FAUCET/credit
 ```
 
-## Run the wasmd node
+## Run wasmd node
 
-```sh
-export WASMD_HOME=$HOME
+TODO persistent peer and seed node?
 
+```sh
+export MONIKER=new_validator
 # initialize wasmd configuration
-wasmd init <moniker> --home $WASMD_HOME # default homedir is $HOME
+wasmd init $MONIKER
 # get the testnets genesis file
-curl https://$RPC_ENDPOINT/genesis | jq .result.genesis > $WASMD_HOME/genesis.json
+curl https://$RPC/genesis | jq .result.genesis > ~/.wasmd/config/genesis.json
 # You need to configure p2p seeds
 # Either you can insert the seed addresses in $HOMEDIR/.wasmd/config/config.toml to "seeds"
 # For simplicity we will pass the seed ID and domain as argument
 # You can get the seed it using command:
-NODE_ID=$(curl https://$RPC_ENDPOINT/status | jq .result.node_info.id | tr -d '"')
+export NODE_ID=$(curl https://$RPC/status | jq .result.node_info.id | tr -d '"')
 
 ## Start wasmd
-wasmd start --p2p.seeds $NODE_ID@$P2P
+wasmd start --p2p.seeds $NODE_ID@$SEED_NODE
 ```
- 
+
 Now you should be seeing blocks being replayed and your node is catching up with the testnet. This could take a while.
 
+## Become an active validator
```

Comment:
You can mention this is optional. The above runs a full node. If they have tokens and have synced, now they can upgrade to a validator